packages/editor/src/lib/editor/managers/TextManager.ts

```diff
@@ -40,12 +40,16 @@ const spaceCharacterRegex = /\s/
 export class TextManager {
 	baseElm: HTMLDivElement
 
+	prevElm?: HTMLDivElement
+	debug = false
```

Comment:
got tired of writing and removing these lines every time I have to debug text measurement, it's blessed now